genshin/utility/uid.py

```diff
@@ -39,18 +51,9 @@ def create_short_lang_code(lang: str) -> str:
 
 def recognize_genshin_server(uid: int) -> str:
     """Recognize which server a Genshin UID is from."""
-    server = {
-        "1": "cn_gf01",
-        "2": "cn_gf01",
-        "5": "cn_qd01",
-        "6": "os_usa",
-        "7": "os_euro",
-        "8": "os_asia",
-        "9": "os_cht",
-    }.get(str(uid)[0])
```

Comment:
keeping this and using `str(uid)[:-8]` would work too but that might be too much python magic.